### PR TITLE
AMLS-4988

### PR DESCRIPTION
**Pull request by drewmiley**
Content changes to Renewal and Submission flow.

## Related / Dependant PRs?

https://github.com/hmrc/amls-frontend/pull/1037

https://github.com/hmrc/amls-frontend/pull/1041/files/f1ac3d7d769a08066f8ca3402b256aeb84a6482e..75cdc6afaf996b25f0f7f5f360941534ad7ccfa5

## How Has This Been Tested?

Manual testing

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/renewal/amls_turnover.scala.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 @headingText = @{
     businessTypes match {
-        case Some(types) if types.length == 1 => "renewal.turnover.title.single.service"
+        case Some(types) if types.length == 1 => "renewal.turnover.title"
```
Comment (drewmiley) [inline app/views/renewal/amls_turnover.scala.html]: Ask about this

File: conf/messages [hunk 2]
```diff
@@ -495,7 +495,7 @@ hint.8.digits = This is 8 numbers, like 12345678, or a mix of 15 letters and num
 hint.9.digits = It’s 9 numbers, like 123456789
 hint.10.digits = For example: 1234567890
 hint.255.chars = Enter 255 characters or less.
```
Comment (drewmiley) [inline conf/messages]: Just use this instead

File: conf/messages [hunk 3]
```diff
@@ -2117,24 +2117,32 @@ progress.renewal.description = Information about your business over the last 12
...
-renewal.percentage.info.line4 = several related cash payments, including payment plans, payments on account, and deposits in any 90 day period
+renewal.percentage.info.line3 = any 90 day period
+renewal.percentage.info.line4 = any single cash payment
+renewal.percentage.info.line5 = several related cash payments, including payment plans, payments on account, and deposits
 
 renewal.business-turnover.title = How much total net profit did your business make in the last 12 months?
-renewal.receiving.title = In the last 12 months did you receive cash payments over €10,000 from customers you’ve not met in person?
+renewal.receiving.title = In the last 12 months did you receive cash payments over €10,000 from customers you have not met in person?
+renewal.receiving.text = If you do not use euros, you’ll need to convert the amount to the currency you use.
+renewal.receiving.info.line1 = Cash can be physical money, notes and coins. It can be paid to you in person or directly into a bank account.
```
Comment (drewmiley) [inline conf/messages]: Line1 to line 5 naming should be changed

File: conf/messages [hunk 4]
```diff
@@ -2117,24 +2117,32 @@ progress.renewal.description = Information about your business over the last 12
...
-renewal.turnover.title = How much total net profit did your business make in the last 12 months from the following services?
-renewal.percentage.turnover.text = If you do not use euros you’ll need to convert the amount to the currency you use.
+renewal.turnover.title = How much total net profit did your business make from services registered for anti-money laundering supervision in the last 12 months?
+renewal.turnover.text = You have registered the following services for anti-money laundering supervision:
+renewal.percentage.turnover.text = If you do not use euros, you’ll need to convert the amount to the currency you use.
 renewal.percentage.info.line1 = Cash can be physical money, notes and coins. It can be paid to you in person or directly into a bank account.
 renewal.percentage.info.line2 = This includes payment of €10,000 or more in:
-renewal.percentage.info.line3 = any single cash payment
-renewal.percentage.info.line4 = several related cash payments, including payment plans, payments on account, and deposits in any 90 day period
+renewal.percentage.info.line3 = any 90 day period
```
Comment (drewmiley) [inline conf/messages]: Line1 to line 5 naming should be changed

File: app/views/estateagentbusiness/business_servicess.scala.html [hunk 5]
```diff
@@ -40,14 +40,14 @@
...
                 @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.05", value = "05")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.02", value = "02")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.08", value = "08")
+                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.06", value = "06")
                 @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.07", value = "07")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.04", value = "04")
-                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.01", value = "01")
+                @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.08", value = "08")
                 @checkbox(f = f("services[]"), labelText = "estateagentbusiness.service.lbl.09", value = "09")
             }
```
Comment (slavvco) [inline app/views/estateagentbusiness/business_servicess.scala.html]: here you've changed the order of checkboxes, are the services then in alphabetical order?

File: app/views/renewal/amls_turnover.scala.html [hunk 6]
```diff
@@ -22,7 +22,7 @@
...
         case _ => "renewal.turnover.title"
     }
```
Comment (slavvco) [inline app/views/renewal/amls_turnover.scala.html]: before it was implemented in a way, that you would see different title based on number of services, now it's showing the same title in both cases, is that right?

File: conf/messages [hunk 7]
```diff
@@ -1090,44 +1093,44 @@ tradingpremises.subheading = Trading premises
...
+estateagentbusiness.service.lbl.05 = Development company
 estateagentbusiness.service.lbl.09 = Social housing provision
 estateagentbusiness.redress.lbl = Redress scheme:
-estateagentbusiness.redress.lbl.01 = The Property Ombudsman Limited
-estateagentbusiness.redress.lbl.02 = Ombudsman Services
-estateagentbusiness.redress.lbl.03 = Property Redress Scheme
+estateagentbusiness.redress.lbl.01 = Ombudsman Services
+estateagentbusiness.redress.lbl.02 = Property Redress Scheme
+estateagentbusiness.redress.lbl.03 = The Property Ombudsman Limited
 estateagentbusiness.redress.lbl.04 = Other
```
Comment (slavvco) [inline conf/messages]: I'm not sure you should change the label's texts... let's say you selected estateagentbusiness.redress.lbl.01 before changes, value 01 was mapped to The Property Ombudsman Limited, you are coming back to your application after changes are deployed, then 01 is mapped to Ombudsman Services... :( unless you have necessary changes in the model mapping

File: conf/messages [hunk 8]
```diff
@@ -1323,15 +1326,15 @@ businessmatching.updateservice.summary.whichfitandproper.count.plural = You’ve
...
 bankdetails.whatyouneed.line_3 = sort codes and account numbers
 bankdetails.whatyouneed.enter_details_hint = Enter the details of all your accounts, one at a time.
-bankdetails.accounttype.title = What kind of bank account is this?
-bankdetails.accounttype.lbl.01 = Personal bank account
-bankdetails.accounttype.lbl.02 = Business bank account belonging to this business
-bankdetails.accounttype.lbl.03 = Business bank account belonging to another business
+bankdetails.accounttype.title = What type of bank account is this?
+bankdetails.accounttype.lbl.03 = Personal bank account
+bankdetails.accounttype.lbl.01 = Business bank account belonging to this business
+bankdetails.accounttype.lbl.02 = Business bank account belonging to another business
```
Comment (slavvco) [inline conf/messages]: same as above with labels

Comment (drewmiley) [reply conf/messages]: Will fix in 4928

File: conf/messages (hunk 7 above)
Comment (drewmiley) [reply conf/messages]: Will fix in 4928

File: app/views/estateagentbusiness/business_servicess.scala.html (hunk 5 above)
Comment (drewmiley) [reply app/views/estateagentbusiness/business_servicess.scala.html]: Will fix in 4928

**Pull request merged**